nx.json

```diff
@@ -15,7 +15,9 @@
       "runner": "@nrwl/nx-cloud",
       "options": {
         "cacheableOperations": ["build", "lint", "test", "e2e"],
-        "accessToken": "M2QwZmQ2YTQtMjFjZi00ZmI4LTk0MDItZjEyOTQ4ZmMwMTUyfHJlYWQtd3JpdGU="
+        "accessToken": "M2QwZmQ2YTQtMjFjZi00ZmI4LTk0MDItZjEyOTQ4ZmMwMTUyfHJlYWQtd3JpdGU=",
+        "canTrackAnalytics": true,
+        "maxParallel": 6
```

Comment:
Question: Should this number and `NX_MAX_PARALLEL` in the build script match?

Reply:
I guess NX_MAX_PARALLEL is for the CI, and maxParallel in nx.json is for our local environment? @LayZeeDK

Reply:
Exactly. To the best of my knowledge, 2 parallel jobs work best in GitHub workflows and 6 is the max beneficial with a powerful local machine.